llvm/test/CodeGen/AArch64/GlobalISel/irtranslator-samesign-mir.mir

```diff
@@ -0,0 +1,48 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+# RUN: llc -mtriple aarch64 -run-pass=aarch64-prelegalizer-combiner -verify-machineinstrs %s -o - | FileCheck %s
```

Comment:
Test name says IRTranslator but this is not testing the irtranslator

Reply:
It is part of the import of the samesign flag. Suggestions for better name?

Reply:
If there's really no combine being tested here, move to llvm/test/CodeGen/MIR/ and name icmp-flags?